src/main/kotlin/apply/application/RecruitmentDtos.kt

```diff
@@ -48,3 +48,58 @@ data class RecruitmentItemRequest(
     @field:NotBlank
     var description: String = ""
 )
+
+data class ApplicationFormSaveRequest(
```

Comment:
항상 고민이 많은 부분이네요. 최근 몇 가지 해외 예제를 보다 보니 '동사 + 명사 + Request' 조합을 보게 되었는데 이거다 싶더라고요. 

- `ActivateTenantCommand`
- `AddGroupToGroupCommand`
- `ChangeUserPasswordCommand`
- `CancelMeetingCommand` 
- ...

```suggestion
data class SaveApplicationFormRequest(
```